torch/_torch_docs.py

```diff
@@ -5135,12 +5135,9 @@ def merge_dicts(*dicts):
 """ + """
 
 .. warning::
-    Not providing a value for :attr:`steps` is deprecated. For backwards
-    compatibility, not providing a value for :attr:`steps` will create a tensor
-    with 100 elements. Note that this behavior is not reflected in the
-    documented function signature and should not be relied on. In a future
-    PyTorch release, failing to provide a value for :attr:`steps` will throw a
-    runtime error.
+    Not providing a value for :attr:`steps` is deprecated. If you are
```

Comment:
"From PyTorch 1.11 linspace requires the `steps` argument. Use `steps=100` to restore the previous behavior."

I think that can be a note instead of a warning, too.